src/main/java/ua/okwine/productexpirationdate/entity/Product.java

```diff
@@ -0,0 +1,117 @@
+package ua.okwine.productexpirationdate.entity;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.*;
+import java.util.Date;
+
+@Entity
+@Table(name="product")
+public class Product {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private int id;
```

Comment:
you should understand that when you're using int as an id you will be restricted by it's max value. It's a common practice to use UUID now, it's unique and the range is much larger. https://github.com/FedosovMax/knight-todo/blob/master/src/main/java/com/knighttodo/knighttodo/gateway/privatedb/representation/Day.java